worlds/default.sdf

```diff
@@ -2,9 +2,9 @@
 <sdf version="1.9">
   <world name="default">
     <physics type="ode">
-      <max_step_size>0.004</max_step_size>
+      <max_step_size>0.002</max_step_size>
       <real_time_factor>1.0</real_time_factor>
-      <real_time_update_rate>250</real_time_update_rate>
+      <real_time_update_rate>500</real_time_update_rate>
```

Comment:
Can you elaborate on how it segfaults and why? Does it segfault on the gz side? or the px4 instance?

Reply:
I tried a lot of things and in the end, this is what worked, it segfaults on the GZ side, I had a similar issue when I was working on a boat sim, and increasing the update rate worked well.

Reply:
I think the reason this happens is because of the steering, when we change the steering angle, part of the wheel goes into the ground, and I believe if the step is too large, it gives room for the force calculation to diverge and explode.

Reply:
Not sure if I understand. Why does this not happen to a drone standing still or colliding with the ground?